lib/manager-tasks/getDeprecatedGroupsTask.ts

```diff
@@ -1,27 +0,0 @@
-import { task, types } from "hardhat/config";
-import { setLocalNodeDeploymentPath, TransactionArguments } from "../helpers/interfaceHelper";
-import {
-  LOG_LEVEL,
-  LOG_LEVEL_DESCRIPTION,
-  MANAGER_GET_DEPRECATED_GROUPS_TASK_DESCRIPTION,
```

Comment:
`MANAGER_GET_DEPRECATED_GROUPS_TASK_DESCRIPTION` can also be removed from `"../helpers/staticVariables"`